inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -464,7 +469,9 @@ def advisors2marc(self, key, value):
     """The advisors for all types of degrees."""
     return {
         'a': value.get('name'),
-        'g': value.get('_degree_type')
+        'g': value.get('_degree_type'),
+        'x': get_recid_from_ref(value.get('record')),
+        'y': '1' if value.get('curated_relation') else None
```

Comment:
Uh? But this record had no `701__y` field... Do you have an example of an Hepname with a `701__y`?